index.html

```diff
@@ -11,7 +11,7 @@
       <h1>HULKs</h1>
       <p>
       <b>RoboCup SPL Team<br />
-        TU Hamburg, Germany</b>
+        TUHH, Germany</b>

```

Comment:
I would keep TU Hamburg to keep it consistent with our current logo. Is there a reason to change this?

Reply:
The reason why I made this change is because TU (Technische Universität) is a German expression and does not exist in the English name of the university (Hamburg University of Technology). Therefore, one could either use the English name (Hamburg University of Technology) which is fairly long, or use the official abbreviation of our university (TUHH).

Reply:
In my personal feeling, TU Hamburg or TUHH feels more "normal" than Hamburg University of Technology, in fact if you directly translate, TU Hamburg is the outcome considering pattern used by TuM, etc (Google and Deepl now show official one). IDK why they have the official translation as it is.